ddtrace/mocktracer/mockspan_test.go

```diff
@@ -176,3 +176,11 @@ func TestSpanFinish(t *testing.T) {
 	assert.True(s.FinishTime().Before(time.Now()))
 	assert.Equal(want, s.Tag(ext.Error))
 }
+
+func TestSpanWithID(t *testing.T) {
+	var spanId uint64 = 123456789
```

Comment:
Let's fix this to make the CI problem go away:
```go
spanID := uint64(123456789)
```

Reply:
fixed